### ISSUE TITLE
There is no images in 'Fate Reforged' 2015 last set

### ISSUE THREAD

**Issue by calamarico** [ontehfritz/api.mtgdb.info#93]
There are only low res images in last set: 'Fate Reforged', an example:

http://api.mtgdb.info/content/card_images/391819.jpeg
http://api.mtgdb.info/content/hi_res_card_images/391819.jpg

Same Resolution.

**Comment by ontehfritz** [comment]
yes there is delay in the his-res images, as it takes awhile to find hi-res scans. Some one has to physical scan the cards when I find a source they will be uploaded. Thank! I will leave this open until they are uploaded

**Comment by calamarico** [comment]
Ok! Thanks!
